moveit_core/robot_model/include/moveit/robot_model/aabb.h

```diff
@@ -0,0 +1,57 @@
+/*********************************************************************
+* Software License Agreement (BSD License)
+*
+*  Copyright (c) 2013, Ioan A. Sucan
```

Comment:
this is non-standard - Ioan worked for Willow at this time so we don't need both Copyright names, just Willow's

Reply:
Ah, missed this one. Should I also update the year at Willow Garage?